human-dynamics-estimator/CMakeLists.txt

```diff
@@ -13,12 +13,10 @@ yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/thrifts/HumanDynamics.thrift" "${CM
 
 add_library(human-dynamics-estimator-data ${THRIFT_SOURCES} ${THRIFT_HEADERS})
 target_include_directories(human-dynamics-estimator-data SYSTEM PUBLIC ${THRIFT_INCLUDE_DIRS} ${YARP_INCLUDE_DIRS})
-target_link_libraries(human-dynamics-estimator-data YARP::YARP_init)
-target_link_libraries(human-dynamics-estimator-data YARP::YARP_OS)
-
+target_link_libraries(human-dynamics-estimator-data YARP::YARP_init YARP::YARP_OS)
 
 ## Include directories
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_include_directories(human-dynamics-estimator-data PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
Nope.. `human-dynamics-estimator-data` does not need to include `${CMAKE_CURRENT_SOURCE_DIR}/include`

It is `human-dynamics-estimator` that needs those files